packages/server/test/e2e/2_browser_path_spec.coffee

```diff
@@ -0,0 +1,48 @@
+path = require("path")
+exec = require("child_process").exec
+
+e2e = require("../support/helpers/e2e")
+Fixtures = require("../support/helpers/fixtures")
+launcher = require("@packages/launcher")
+
+absPath = (pathStr) ->
```

Comment:
I'm going to approve this PR - but this feels like this is something that should be part of the found browsers objects in `launcher`. 

It looks like we have a `path` property... but apparently that only returns the binary executable and not the actual path to it?